src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -680,7 +609,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             onAnnotationSelect,
         } = this.props;
         const currentFileVersionId = getProp(file, 'file_version.id');
-        const match = getAnnotationsMatchPath(history);
+        const match = getAnnotationsMatchPath(history.location);
```

Comment:
Is `history.location` and `location` always the same thing? It looks like we use the former here, but the latter elsewhere. Should we standardize?

Reply:
Should be same, I'll change it to be consistent